src/content/ccip/architecture.mdx

```diff
@@ -149,6 +149,8 @@ Users should note that CCIP fees include both gas cost overhead and a flat/basis
 
 Current CCIP premium fees are in line with industry standards within the cross-chain ecosystem, but these values are subject to change. The premium portion of fees paid in alternative assets will have a surcharge of 10% when compared to LINK payments. To understand service limits, read the [CCIP Service Limits](/ccip/supported-networks#service-limits) page.
 
+Aside from billing, overhead costs, and fees, remember to [carefully estimate the `gasLimit` that you set](/ccip/best-practices#setting-gaslimit) for your destination contract so CCIP can have enough gas to execute `ccipReceive()`, if applicable. Any unspent gas from this user-set limit is not refunded.
```

Comment:
Similarly to VRF, the gas limit you set here isn't really considered part of the service's billing model itself, so I assume that's why it wasn't listed here. 

But it IS a part of the overall cost of using the service (where, as I understand it, total cost = gas overheads, fees, gas limits, and premiums that are specified by the billing model). That's why it would be better to have a separate page about costs that explains all these pieces in the same place, similar to the VRF Estimating Costs page. Even before having a service-specific "cost estimating calculator" for CCIP it would be beneficial to have all the info in one place.